doc/make.bat

```diff
@@ -0,0 +1,45 @@
+@ECHO OFF

```

Comment:
Should we even provide this file? It works for building the documentation on Windows, but without regular testing, I'm worried that this will just rot.

Reply:
336b0244be26